### PR TITLE
Added solution for the 4th task in check.io

### PR DESCRIPTION
**Pull request by kosenkoNick**
image

### CODE REVIEW

File: string_into_pairs/splitter_zip.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    - 'abcdef' -> zip('ace', 'bdf_') -> ['ab', 'cd', 'ef'] - len(string) is even, so zip will make 3 pairs, '_' is redundant
+    - 'abcdef' -> zip('ace', 'bd_') -> ['ab', 'cd', 'e_']  - len(string) is odd, so '_' will make lengths equal, and it will be used
+
+    :param string: string to split
+    :return: iterable of strings
+    """
+    return [ch1 + ch2 for ch1, ch2 in zip(string[::2], string[1::2] + '_')]
+
+
+print(split_pairs('abcdef'))
```
Comment (fkudinov-mn) [inline string_into_pairs/splitter_zip.py]: remove line

File: string_into_pairs/splitter_range.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+def split_pairs(string: str) -> Iterable[str]:
+    """
+    Method splits given string into pairs. If string length is odd - it adds '_' symbol
+    :param string: string to split
+    :return: iterable of strings
+    """
+    if len(string) % 2 == 1:
+        string += '_'
+
+    return [str(string[i:i+2]) for i in range(0, len(string), 2)]
```
Comment (fkudinov-mn) [inline string_into_pairs/splitter_range.py]: code style: spaces around operator

File: string_into_pairs/splitter_range.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+
+def split_pairs(string: str) -> Iterable[str]:
+    """
+    Method splits given string into pairs. If string length is odd - it adds '_' symbol
+    :param string: string to split
+    :return: iterable of strings
+    """
+    if len(string) % 2 == 1:
+        string += '_'
```
Comment (fkudinov-mn) [inline string_into_pairs/splitter_range.py]: string =  string + '_' if  len(string) % 2 else string

File: string_into_pairs/splitter_zip.py (hunk 1 above)
Comment (kosenkoNick) [reply string_into_pairs/splitter_zip.py]: Done

File: string_into_pairs/splitter_range.py (hunk 2 above)
Comment (kosenkoNick) [reply string_into_pairs/splitter_range.py]: Done

**Pull request merged**